Samuel changes to master branch

samuelhornick81 opened the pull request.

samuelhornick81 committed:
little change for now

I added a view for managing the students and faculty for the admin, and added things so the buttons will work. I will work on more later tonight.

samuelhornick81 committed:
Theme changed

I changed the theme from solar back to the default bootstrap theme because I think it makes the website look more in line with other USI websites. I plan to work more over the weekend and probably make actual progress.

Gage-cook committed:
Updated CSS styling to match the required design.

samuelhornick81 committed:
added a chat log page

samuelhornick81 committed:
Merge pull request #61 from jlarnett/master

master changes to my branch

Gage-cook committed:
Changes

Changes

samuelhornick81 committed:
Admin Tab switch

samuelhornick81 committed:
CSS fixes

samuelhornick81 committed:
Merge pull request #68 from jlarnett/master

Updating Samuel branch with master changes

samuelhornick81 committed:
small change

samuelhornick81 committed:
Merge pull request #71 from jlarnett/master

adding master stuff to my branch before I do push mine into master

samuelhornick81 committed:
Solar theme back for now

Gage-cook committed:
The new pages

Gage-cook committed:
yoooo

jlarnett commented on Campus_SMS/appsettings.json:
@aholmes100 heads up I think this key has been exposed now. @samuelhornick81 we don't want to add secrets to this json file. It is shown in source control and means in real world you would have to cycle out the keys.

samuelhornick81 committed:
Update appsettings.json

oops

jlarnett committed:
populated adminDashboardViewModel with data from database.

samuelhornick81 committed:
Fixed build error

there was a line at the top of Index-Gage causing a build error so I removed it.

samuelhornick81 committed:
Lot of work on the Admin faculty tab

samuelhornick81 committed:
Merge pull request #78 from jlarnett/master

master to samuel again

samuelhornick81 committed:
Student Tab

samuelhornick81 committed:
Remove buttons

You can remove phone numbers and faculty from classes on the admin page and some CSS changes

mmahoney09110 committed:
Merge branch 'master' into Samuel

samuelhornick81 committed:
fixed admin controller
